app/code/Magento/Customer/Controller/Address/FormPost.php

```diff
@@ -89,18 +96,44 @@ public function __construct(
             $dataProcessor,
             $dataObjectHelper,
             $resultForwardFactory,
-            $resultPageFactory
+            $resultPageFactory,
+            $customerRepository
         );
     }
 
     /**
      * Extract address from request
      *
-     * @return \Magento\Customer\Api\Data\AddressInterface
+     * @return AddressInterface
+     * @throws \Exception
+     * @deprecated Use extractCurrentAddress instead
+     */
+    protected function _extractAddress(): AddressInterface
+    {
+        return $this->extractCurrentAddress();
+    }
+    /**
+     * Retrieve existing address data
+     *
+     * @return array
+     * @throws \Exception
+     * @deprecated Use _getExistingAddressData instead
+     */
+    protected function getExistingAddressData(): array
```

Comment:
Please, make this method private

Reply:
the same here, `getExistingAddressData` is original one `_getExistingAddressData` was added